app-index.json

```diff
@@ -39,5 +39,26 @@
   "com.ToastVR.RichiesPlankExperience":{"name": "Richies Plank Experience","icon":"app-index-icons/richiesplankexperience.png"},
   "com.steelcrategames.keeptalkingandnobodyexplodes":{"name": "Keep talking and nobody explodes","icon":"app-index-icons/keeptalking.png"},
   "com.picselica.OceanRiftFullQuest":{"name": "Ocean Rift","icon":"app-index-icons/oceanrift.png"},
+  "com.sidequest.launcher":{"name": "SideQuest Launcher","icon":"app-index-icons/sidequest.png"},
+  "com.YourCompany.RoboRecall":{"name": "Robo Recall","icon":"app-index-icons/roborecall.png"},
+  "com.resolutiongames.abvriop.santacruz":{"name": "Angry Birds VR: Isle of Pigs","icon":"app-index-icons/angrybirdsvriof.png"},
+  "com.coatsink.nexus":{"name": "Shadow Point","icon":"app-index-icons/shadowpoint.png"},
```

Comment:
Looks like this is the wrong packagename maybe? Its also on Apex Construct below
`"com.coatsink.nexus":{"name": "Apex Construct","icon":"app-index-icons/apexconstruct.png"},`

Reply:
The apex construct package was wrong. It should now be corrected as "com.fasttravelgames.apexconstruct"